lib/aggregate-browsers.js

```diff
@@ -1,29 +1,36 @@
 'use strict'
 
-const _ = require('lodash')
+function aggregate (obj) {
```

Comment:
Nit: `obj` is an array but its name suggests otherwise. Perhaps rename to `arr`.